### PR TITLE
Add initial definition for the Switch Cluster and activate it for some configurations

### PR DESCRIPTION
**Pull request by vivien-apple**
#### Problem

This PR add the initial definition for the `Switch Cluster`, ignoring the `events` parts until it is supported by IM but instead rely on reporting in the meantime.

 #### Summary of Changes
 * Add Initial definition into `src/app/zap-templates/switch-cluster.xml`
 * Activate it into a bunch of `.zap` files
 * Update `gen/` folder

### CODE REVIEW

**Review by msandstedt — APPROVED** [review]

**Review by woody-apple — APPROVED** [review]

**Comment by woody-apple** [comment]
@saurabhst @mspang @andy31415 ?

**Comment by woody-apple** [comment]
@Damian-Nordic @jepenven-silabs

**Comment by woody-apple** [comment]
@hawk248 ?

File: src/app/zap-templates/templates/chip/python-CHIPClusters-cpp.zapt [hunk 1]
```diff
@@ -71,7 +71,7 @@ void chip_ime_SetFailureResponseDelegate(FailureResponseDelegate delegate)
     gFailureResponseDelegate = delegate;
 }
 
-{{#chip_client_clusters}}
+{{#chip_server_clusters}}
```
Comment (mspang) [inline src/app/zap-templates/templates/chip/python-CHIPClusters-cpp.zapt:74]: Is this intended?

**Review by mspang — APPROVED** [review]

**Pull request merged**